data/data/gcp/network/firewall-compute.tf

```diff
@@ -7,7 +7,7 @@ resource "google_compute_firewall" "compute_ingress_icmp" {
   }
 
   source_ranges = [var.network_cidr]
-  target_tags   = ["${var.cluster_id}-worker"]
+  target_tags   = ["${var.cluster_id}-compute"]
```

Comment:
https://github.com/openshift/installer/pull/1938/commits/c11f2d2a72f3016842c9e50f2a866a10018d1f3b

needs to stay master and worker